codice/ddf
# DDF-2258 Update the JSON definition file parser to handle updates and…

#### What does this PR do?
Updates the JSON definition file parser to handle file updates and deletes.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@rzwiefel 
@coyotesqrl 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jlcsmith
@kcwire

#### How should this be tested?
Drop one or more JSON definition files into `etc/definitions`. Try modifying and deleting some of the definition files and verify that the expected changes are applied by ingesting or updating metacards.

#### What are the relevant tickets?
[DDF-2258](https://codice.atlassian.net/browse/DDF-2258)

#### Checklist:
